modules/ROOT/pages/flow-counting-studio.adoc

```diff
@@ -5,13 +5,20 @@ endif::[]
 
 In Anypoint Studio 7.20 and later, you can track the number of billable flows in your project.
 
-include::anypoint-code-builder::partial$acb-studio-shared-flow-counting.adoc[tag="flow-counting-intro"]
+Flow Counting improves transparency in usage-based pricing by providing visibility into the number of billable flows when developing your project. With this new feature, you better understand your consumption patterns from design to deployment, preventing unexpected billing issues.
 
-include::anypoint-code-builder::partial$acb-studio-shared-flow-counting.adoc[tag="flow-counting-billable-flows"]
+Your Anypoint Platform organization pays for billable flows, such as Trigger Flows or APIkit-generated flows, that are currently deployed and running in your project. To reduce these costs, consider using subflows or private flows, which aren't billable. Additionally, to bypass charges for unused flows, avoid deploying unnecessary flows from accelerators or assets. See xref:general::pricing.adoc[] for information about managing and reducing billable flows.
```

Comment:
I'd change 'in your project' to 'in your runtime plane'.

Reply:
OK! Thanks!